python/.flake8

```diff
@@ -0,0 +1,8 @@
+[flake8]
+# Pinterest standard line length.
+max-line-length=120
```

Comment:
Black defaults to 88(-ish) columns per line, so this flake8 setting isn't going to do much.

- https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length

Reply:
It seems to be doing something, because flake8 flags 284 errors without this configuration. I'm not sure why black isn't enforcing line-length, because it rewrote pretty much all of the files.
```
$ flake8 -qq --statistics
284   E501 line too long (84 > 79 characters)
$ flake8 | grep 'E501 line too long' | sed 's/(//' | sed 's/)//' | awk '{print $6,$7,$8,$9}' | sort -n | uniq -c
  31 80 > 79 characters
  25 81 > 79 characters
  30 82 > 79 characters
  23 83 > 79 characters
  27 84 > 79 characters
  15 85 > 79 characters
  25 86 > 79 characters
  19 87 > 79 characters
  19 88 > 79 characters
   3 89 > 79 characters
   7 90 > 79 characters
   3 91 > 79 characters
   4 92 > 79 characters
   7 93 > 79 characters
   6 94 > 79 characters
   3 95 > 79 characters
   4 96 > 79 characters
   2 97 > 79 characters
   3 98 > 79 characters
   1 99 > 79 characters
   4 100 > 79 characters
   5 101 > 79 characters
   3 102 > 79 characters
   2 103 > 79 characters
   5 104 > 79 characters
   1 105 > 79 characters
   1 106 > 79 characters
   2 108 > 79 characters
   2 112 > 79 characters
   1 119 > 79 characters
   1 120 > 79 characters
```

Reply:
I'm not able to guess unless I see an example of a line that is coming in at 90+.

... but also, black's approach is "try to respect --line-length, but don’t become crazy if you can’t", per https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length.

Reply:
Got it. Line length adjusted uniformly to 88. Most were comments, long documentation URLs, and output messages.